website_docs/getting-started.md

```diff
@@ -127,11 +127,14 @@ func main() {
 		return
 	case err := <-errCh:
 		if err != nil {
-			l.Fatal(err)
+			l.Panic(err)
```

Comment:
Personally, I favor using a non-panicking pattern and use the `defer func() { os.Exit(exitCode) }()` pattern (method 2 in https://www.golinuxcloud.com/golang-handle-defer-calls-os-exit/)